shell/platform/embedder/embedder.h

```diff
@@ -1974,7 +1976,7 @@ typedef const FlutterLocale* (*FlutterComputePlatformResolvedLocaleCallback)(
     size_t /* Number of locales*/);
 
 typedef struct {
-  /// This size of this struct. Must be sizeof(FlutterDisplay).
+  /// The size of this struct. Must be sizeof(FlutterDisplay).
```

Comment:
Looks like the comment was wrong in at least two ways!
```suggestion
  /// The size of this struct. Must be sizeof(FlutterEngineDisplay).
```